### PR TITLE
tests

### PR DESCRIPTION
**Pull request by ramazanix**

### CODE REVIEW

File: db/migrate/20210923132042_create_tasks.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def change
       t.text :description
       t.datetime :deadline_at
       t.references :project, null: false, foreign_key: true
-
+      t.references :user, null: false, foreign_key: true
```
Comment (DmitryBarskov) [inline db/migrate/20210923132042_create_tasks.rb:8]: Where is the migration file?

File: spec/policies/task_policy_spec.rb [hunk 2]
```diff
@@ -3,46 +3,41 @@
 RSpec.describe 'TaskPolicy', type: :policy do
   subject(:policy) { TaskPolicy.new(user, task) }
 
+  let(:task) {Task.new}
```
Comment (DmitryBarskov) [inline spec/policies/task_policy_spec.rb]: Why do you need it here? This variable seems to be unused.

File: spec/policies/task_policy_spec.rb [hunk 3]
```diff
@@ -3,46 +3,41 @@
...
+
+      it { is_expected.to eq(true) }
+    end
   end
 
   describe '#create?' do
     subject { policy.create? }
 
-    let(:project) { Project.new(id: 42, user: creator) }
+    let(:project) { Project.new(id: 1, user: creator) }
```
Comment (DmitryBarskov) [inline spec/policies/task_policy_spec.rb:27]: Why do you think the project need an ID?

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

File: db/migrate/20210923132042_create_tasks.rb (hunk 1 above)
Comment (ramazanix) [reply db/migrate/20210923132042_create_tasks.rb:8]: Я просто не включил его в коммит. Сейчас прикреплю

File: db/migrate/20211114191403_add_user_to_task.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class AddUserToTask < ActiveRecord::Migration[6.1]
+  def change
+    change_table :tasks do |table|
+      table.bigint :user_id, null: true
+
+      table.index :user_id
```
Comment (DmitryBarskov) [inline db/migrate/20211114191403_add_user_to_task.rb:6]: you should add contstraints.

File: spec/features/tasks/delete_spec.rb [hunk 5]
```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+RSpec.describe 'Delete task', type: :feature do
```
Comment (DmitryBarskov) [inline spec/features/tasks/delete_spec.rb:2]: Rewrite this file from scratch. You must not copy some other code.

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**